docs/source/_toctree.yml

```diff
@@ -25,6 +25,7 @@
     title: Summary of the tokenizers
   - local: multilingual
     title: Multi-lingual models
+
```

Comment:
Not sure this new line is intentional. It shouldn't be needed.

Reply:
Not sure where that came from - deleted.